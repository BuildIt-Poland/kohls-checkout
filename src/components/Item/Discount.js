import styled from 'styled-components';

import { colorRed, fontWeightBold } from '../../styles/designTokens';
import SmallText from './SmallText';

const Discount = styled(SmallText)`
  color: ${colorRed};
  font-weight: ${fontWeightBold};
`;

export default Discount;
