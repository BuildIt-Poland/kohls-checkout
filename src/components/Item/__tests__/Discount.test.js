import React from 'react';
import { create } from 'react-test-renderer';

import Discount from '../Discount';

describe('COMPONENT - Item Discount', () => {
  it('renders correctly', () => {
    const component = create(<Discount />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
