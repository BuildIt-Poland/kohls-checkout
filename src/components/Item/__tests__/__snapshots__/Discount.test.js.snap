// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`COMPONENT - Item Discount renders correctly 1`] = `
.c0 {
  font-size: 1.4rem;
  font-weight: 400;
  color: #222222;
  margin: 0;
  -webkit-text-decoration: none;
  text-decoration: none;
  color: #e0151d;
  font-weight: 700;
}

<p
  className="c0"
/>
`;
